logic/account.js

```diff
@@ -508,24 +510,24 @@ class Account {
 						}
 
 						// If updated value is positive number
-						if (value.greaterThan(0)) {
+						if (value.isGreaterThan(0)) {
 							promises.push(
 								dbTx.accounts.increment(
 									address,
 									updatedField,
-									value.floor().toString()
+									value.toFixed().toString()
```

Comment:
Possible change of behavior here, it should be `value.integerValue(Bignum.ROUND_FLOOR).toFixed()` or even better `value.toFixed(0, Bignum.ROUND_FLOOR)`. Are you sure that we always have integer here? If yes - we don't need `.toFixed()` here (or `toString()`, depends on preference).